.github/actions/setup-local-volume-provisioner/action.yaml

```diff
@@ -1,37 +1,32 @@
 name: "Setup Local Volume Provisioner"
 description: 'Installs Local Volume Provisioner'
-inputs:
-  repositoryPath:
-    description: "Path to where the project code lives"
-    required: true
-  numberOfPersistentVolumes:
-    description: "Number of persistent volumes that will be provisioned"
-    required: true
 runs:
   using: "composite"
   steps:
-  - name: Prepare host directories
+
+  - name: Checkout Dynamic Local Volume Provisioner
+    uses: actions/checkout@v3
+    with:
+      repository: scylladb/k8s-local-volume-provisioner
+      path: k8s-local-volume-provisioner
+  
+  - name: Create XFS volume
     shell: bash
     run: |
       set -euExo pipefail
       shopt -s inherit_errexit
       
-      sudo mkdir /mnt/persistent-volumes
-      # /mnt has limited space in GH Actions, use /
-      sudo mkdir /persistent-volumes
-      sudo mount --bind /persistent-volumes /mnt/persistent-volumes
```

Comment:
shouldn't this stay?

Reply:
done, not on `/` but `/var` holds the disk image which is on the same device.